<launch>
  
  <node name="vrmapping" pkg="vrmapping" type="vrmapping_node" output="screen">
  <rosparam file="$(find vrmapping)/config/parameters.yaml" command="load" />
  </node>
  <include file="$(find vrmapping_ui)/launch/vrm_ui_jy.launch"/>
</launch>